src/playbooks/with_build/vsc_ha_upgrade_deploy_1.yml

```diff
@@ -1,4 +1,4 @@
----
+  ---
```

Comment:
Looks like this may have been an accident, please revert this change so this file isn't included in the PR

Reply:
Done!